modules/integration/tests-integration/tests-amqp/src/test/java/org/wso2/mb/integration/tests/amqp/functional/HierarchicalTopicsTestCase.java

```diff
@@ -148,10 +152,34 @@ public void performHierarchicalTopicsImmediateChildrenTestCase()
                    AndesClientException, XPathExpressionException {
 
         // Creating clients
+        AndesClient consumerClient1 = getConsumerClientForTopic("games.*");
+        consumerClient1.startClient();
+
+        AndesClient publisherClient1 = getPublishingClientForTopic("games");
+        publisherClient1.startClient();
+
+        AndesClientUtils
+                .waitForMessagesAndShutdown(consumerClient1, AndesClientConstants.DEFAULT_RUN_TIME);
+
```

Comment:
The new lines are added in these to separate the consumer and publisher in each test scenario, which I think gives better readability here.